hil/cli.py

```diff
@@ -37,7 +39,7 @@
 usage_dict = {}
 MIN_PORT_NUMBER = 1
 MAX_PORT_NUMBER = 2**16 - 1
-
+VERSION = pkg_resources.require('hil')[0].version
```

Comment:
any reason why you are getting version here and not in the function itself?

Reply:
I use Ian's code as an example:https://github.com/zenhack/simp_le/blob/0fc118de6c0fcad0e7a23a8e788a1e00995a1f5a/simp_le.py#L67

And I have seen people coding this way in other places. I guess it is a convention? But I understand we only use this once in the entire code base. Maybe using just the function makes sense?

Reply:
Don't feel too strongly about this. If it's a convention which might enable some reuse in the future, I am okay with it.

Reply:
hmm, @zenhack might know better here. I am not sure whether this is kinda a convention but it seems like so to me :)

Reply:
I don't have a strong opinion either. The code I linked to is in a project I inherited; I wasn't around for the decision on that one.

Re: output format, maybe just something like:

```
HIL version ${version_number}
```

You sound make sure to add a newline, so it doesn't end up on the same line as your prompt.

Reply:
Sure, was planning to add that once I have an idea about how the output should look like. Should've mentioned that earlier. Thanks.